pkg/pipelines/triggers/bindings.go

```diff
@@ -52,33 +37,21 @@ func createDevCIBuildBinding(ns string) triggersv1.TriggerBinding {
 	}
 }
 
-func createStageCDDeployBinding(ns string) triggersv1.TriggerBinding {
+func createPushBinding(ns string) triggersv1.TriggerBinding {
 	return triggersv1.TriggerBinding{
 		TypeMeta:   triggerBindingTypeMeta,
-		ObjectMeta: meta.ObjectMeta(meta.NamespacedName(ns, "stage-cd-deploy-from-push-binding")),
+		ObjectMeta: meta.ObjectMeta(meta.NamespacedName(ns, "cd-deploy-from-push-binding")),
 		Spec: triggersv1.TriggerBindingSpec{
 			Params: []pipelinev1.Param{
 				createBindingParam("gitref", "$(body.ref)"),
+				createBindingParam("gitreftag", "$(body.head_commit.id)"),
```

Comment:
I think we can possibly do away with this, and change `gitsha` to use `body.head_commit.id` too.

Might wanna check this with a hook notification.

Reply:
Using `gitsha` with value `body.head_commit.id` in github-push-binding